Dockerfile

```diff
@@ -3,8 +3,15 @@ FROM python:3.11
 WORKDIR /usr/src/app
 
 RUN curl -sSL https://install.python-poetry.org | python3 -
+RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v25.0-rc1/protoc-25.0-rc-1-linux-x86_64.zip
+RUN unzip protoc-25.0-rc-1-linux-x86_64.zip -d protoc
```

Comment:
the rc means that this is a Release Candidate, generally we should avoid not production ready software